recipes/google-compute-engine/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "google-compute-engine" %}
+{% set version = "2.8.13" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 358363a10169f890bac78cf9d7105132cdd2c1546fa8d9caa35c04a7cf7cfba7
+
+build:
+  number: 0
+  entry_points:
+    - google_accounts_daemon=google_compute_engine.accounts.accounts_daemon:main
+    - google_clock_skew_daemon=google_compute_engine.clock_skew.clock_skew_daemon:main
+    - google_instance_setup=google_compute_engine.instance_setup.instance_setup:main
+    - google_network_daemon=google_compute_engine.networking.network_daemon:main
+    - google_metadata_script_runner=google_compute_engine.metadata_scripts.script_manager:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [win]
+
+requirements:
+  host:
+    - boto
```

Comment:
Are these all needed in `host`? I suspect you only need `python`, `pip`.

Reply:
I'll check

Reply:
Yes, they are needed.